src/sentry/hybridcloud/migrations/0005_add_api_and_org_token_replicas.py

```diff
@@ -0,0 +1,139 @@
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0571_add_hybrid_cloud_foreign_key_to_slug_reservation"),
+        ("hybridcloud", "0004_add_cache_version"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ApiTokenReplica",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                (
+                    "scopes",
+                    bitfield.models.BitField(
+                        [
+                            "project:read",
+                            "project:write",
+                            "project:admin",
+                            "project:releases",
+                            "team:read",
+                            "team:write",
+                            "team:admin",
+                            "event:read",
+                            "event:write",
+                            "event:admin",
+                            "org:read",
+                            "org:write",
+                            "org:admin",
+                            "member:read",
+                            "member:write",
+                            "member:admin",
+                        ],
+                        default=None,
+                    ),
+                ),
+                ("scope_list", sentry.db.models.fields.array.ArrayField(null=True)),
+                (
+                    "application_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.ApiApplication", db_index=True, null=True, on_delete="CASCADE"
+                    ),
+                ),
+                (
+                    "organization_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.Organization", db_index=True, null=True, on_delete="SET_NULL"
+                    ),
+                ),
+                ("application_is_active", models.BooleanField(default=False)),
```

Comment:
Does this field need to be kept? I didn't see it in use by the authentication adapters.